drivers/espi/Kconfig

```
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 menuconfig ESPI
-	bool "ESPI Driver"
+	bool "Enhanced Serial Peripheral Interface Bus (eSPI) drivers"
```

For consistency with I2C change proposed below.

```suggestion
	bool "Enhanced Serial Peripheral Interface (eSPI) bus drivers"
```

---

Fixed, thank you.